src/tools/fuzzing/fuzzing.cpp

```diff
@@ -49,7 +49,45 @@ TranslateToFuzzReader::TranslateToFuzzReader(Module& wasm,
     loggableTypes.push_back(Type::v128);
   }
 
+  // Setup params. Start with the defaults.
   globalParams = std::make_unique<FuzzParamsContext>(*this);
+
+  // Some of the time, adjust parameters based on the size, e.g. allowing more
+  // heap types in larger inputs, etc.
+  if (random.oneIn(2)) {
+    // A typical random wasm input from fuzz_opt.py is fairly large (to minimize
+    // the process creation overhead of all the things we run from python), and
+    // the defaults are tuned to that.
+    double size = random.remaining();
+    const double MEAN_SIZE = 40 * 1024;
```

Comment:
Is this number taken from measurements? Do we expect it to change much over time? (Not that it would matter much if it did.)

Reply:
This corresponds to the mean size from `fuzz_opt.py`. I added a comment.